actioncable/app/javascript/actioncable/connection_monitor.js

```diff
@@ -0,0 +1,129 @@
+// Responsible for ensuring the cable connection is in good health by validating the heartbeat pings sent from the server, and attempting
+// revival reconnections if things go astray. Internal class, not intended for direct user manipulation.
+import { log } from "./helpers"
+
+let now = undefined
+let secondsSince = undefined
+let clamp = undefined
```

Comment:
No need to start these out `undefined` and then set them later (immediately, basically).